src/geometryRTheta/time_solver/bsl_predcorr_second_order_implicit.hpp

```diff
@@ -405,12 +390,7 @@ class BslImplicitPredCorrRTheta : public ITimeSolverRTheta
 
             // Compute the new advection field A(X^n) + A(X^{k-1}):
             ddc::for_each(grid, [&](IdxRTheta const irp) {
-                ddcHelper::get<X>(advection_field_k_tot)(irp)
-                        = ddcHelper::get<X>(advection_field)(irp)
-                          + ddcHelper::get<X>(advection_field_k)(irp);
-                ddcHelper::get<Y>(advection_field_k_tot)(irp)
-                        = ddcHelper::get<Y>(advection_field)(irp)
-                          + ddcHelper::get<Y>(advection_field_k)(irp);
+                advection_field_k_tot(irp) = advection_field(irp) + advection_field_k(irp);
```

Comment:
Before this PR this line compiled. However no values were modified. Rather the temporary vector created in `operator()` was modified but did not affect the values saved in `advection_field_k_tot`